.husky/_/pre-commit

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env sh
-npm run pre-commit
+# npm run pre-commit
```

Comment:
This was added in here, but I think it's fine to go with this PR. We can circle back and fix this in https://github.com/dearborn-coding-club/website-base-frontend/issues/39